set.mm

```diff
@@ -209014,21 +209014,20 @@ be used in an extensible structure (slots must be positive integers).
                   A. x e. ( Base ` g )
                   ( ( e ( +g ` g ) x ) = x /\ ( x ( +g ` g ) e ) = x ) ) ) ) $.
 
-    $( Define the group sum (also called "iterated sum") for the structure
-       ` G ` of a finite sequence of elements whose values are defined by the
-       expression ` B ` and whose set of indices is ` A ` .  It may be viewed
-       as a product (if ` G ` is a multiplication), a sum (if ` G ` is an
-       addition) or any other operation.  The variable ` k ` is normally a free
-       variable in ` B ` (i.e., ` B ` can be thought of as ` B ( k ) ` ).  The
-       definition is meaningful in different contexts, depending on the size of
-       the index set ` A ` and each demanding different properties of ` G ` .
+    $( Define a finite group sum (also called "iterated sum") of a structure.
+       Given ` G gsum F ` where ` F : A --> ( Base `` G ) ` , the set of
+       indices is ` A ` and the values are given by ` F ` at each index.  The
+       group sum may be viewed as a product (if ` G ` is a multiplication), a
```

Comment:
```suggestion
       group sum may be viewed as a product (if ` G ` is a multiplicative group), a
```

It looks like the concept of "additive group" exists. Intuitive enough, but perhaps alternatively the sentence could be shortened:

"A group sum over a multiplicative group can be viewed as a product."